packages/shared/components/NodeConfigurationForm.svelte

```diff
@@ -54,14 +53,20 @@
     $: networkId, coinType, node.url, (formError = '')
     $: node = {
         url: node.url,
-        auth: {
-            ...([username, password].every((val) => val !== '') && {
-                basicAuthNamePwd: [username, password],
-            }),
-            ...(jwt !== '' && {
-                jwt,
-            }),
-        },
+        auth: getAuth(),
+    }
+    function getAuth(): IAuth {
+        const auth: IAuth = {}
+        const isBasicAuth = (): boolean => [username, password].every((value) => value !== '')
```

Comment:
question: why is this a function?

Reply:
there's no need for this, but I believe you will understand better this way... my fault, I removed it :pray: